pyanaconda/ui/tui/spokes/__init__.py

```diff
@@ -135,20 +145,21 @@ def prompt(self, entry = None):
             pw = self._app.raw_input(_("%s: ") % entry.title, hidden=True)
             confirm = self._app.raw_input(_("%s (confirm): ") % entry.title, hidden=True)
 
-            if (pw and not confirm) or (confirm and not pw):
-                print(_("You must enter your root password and confirm it by typing"
-                        " it a second time to continue."))
-                return None
```

Comment:
Are you sure that we want to loose this message?

Reply:
I would also expect it to be generally useful - unless the code is unreachable or something similar (that might be possible after the substantial changes we did on password checking a while back).

Reply:
If we want to keep this message, then I will have to change it anyway, because "root password" is hard coded, so for user passwords and passphrases is the text the same.

Now you will see a message: "The passwords you entered were different.  Please try again." I think that is clear what was wrong and what the user should do.